src/server/tracing.ts

```diff
@@ -55,22 +61,27 @@ export async function nodeSDKBuilder() {
   const sdk = new NodeSDK({
     textMapPropagator: new AWSXRayPropagator(),
     instrumentations: [
-      getNodeAutoInstrumentations(),
-      //add instrumentation library below if you want to configure params
       new AwsInstrumentation({
         suppressInternalInstrumentation: true,
       }),
-      new HttpInstrumentation({
-        ignoreIncomingPaths: ['/.well-known/apollo/server-health'],
-      }),
+      new AwsLambdaInstrumentation({}),
+      new DataloaderInstrumentation({}),
       new ExpressInstrumentation({
         ignoreLayersType: [ExpressLayerType.MIDDLEWARE],
       }),
+      new FsInstrumentation({}),
```

Comment:
nit: we also removed fs from parser  as it was producing a lot of noise readings between resolver and sql queries metrics

Reply:
looking now, will remove if not using fs